public/doc/swagger-2.yaml

```diff
@@ -311,16 +311,18 @@ paths:
       parameters:
         - $ref: '#/parameters/PortfolioItemID'
       responses:
-        204:
+        '204':
           description: Portfolio Item deleted
-        404:
+        '404':
```

Comment:
Same as above

Reply:
@syncrou the swagger editor is doing that formatting, when you save the doc as YAML.